frontend/www/js/omegaup/components/problem/CollectionDetails.vue

```diff
@@ -1,20 +1,76 @@
 <template>
   <div>
     <h1 class="card-title">{{ title }}</h1>
+    <div class="row">
+      <div class="col col-md-4">
+        <div>
+          <omegaup-collection-filter-tags
+            :collection="checkedTags"
+            :another-tags="data.anotherTags"
+            :another-tags-displayed="anotherTags"
+            @emit-add-tag="addTag"
+          ></omegaup-collection-filter-tags>
+          <input name="checked_tags" :value="checkedTagsList" type="hidden" />
+          <input
+            name="another_checked_tags"
+            :value="checkedAnotherTagsList"
+            type="hidden"
+          />
+        </div>
+      </div>
+    </div>
   </div>
 </template>
 
 <script lang="ts">
 import { Vue, Component, Prop } from 'vue-property-decorator';
+import collection_Filter_Tags from './FilterTags.vue';
```

Comment:
```suggestion
import problem_FilterTags from './FilterTags.vue';
```

el patrón es `nombredeldirectorio_NombreDelArchivo`.